source/eventHandler.py

```diff
@@ -95,7 +96,10 @@ def __init__(self, eventName, obj, kwargs):
 
 	def next(self):
 		func, args = next(self._gen)
-		return func(*args, **self.kwargs)
+		try:
+			return func(*args, **self.kwargs)
+		except TypeError:
+			return extensionPoints.callWithSupportedKwargs(func, *args, **self.kwargs)

```

Comment:
Please log a clear message here  (perhaps warning) so that we catch these issues pretty quick and fix them.